packages/cf.js/src/utils/signature.ts

```diff
@@ -49,6 +49,10 @@ export class Signature {
     return signaturesList;
   }
 
+  public static fromEthersSignature(sig: ethers.utils.Signature): Signature {
+    return new Signature(sig.recoveryParam as number, sig.r, sig.s);
+  }
```

Comment:
Wait, there's an `ethers.utils.Signature`? Then I would recommend we strictly use this class versus re-inventing our own.

Reply:
Yeah, that's what `signDigest()` returns. The only annoying thing about it is it doesn't have a method to convert to bytestring.